src/annotator/test/annotation-counts-test.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+var annotationCounts = require('../annotation-counts');
+
+describe('annotationCounts', function () {
+  var countEl1;
+  var countEl2;
+  var CrossFrame = null;
```

Comment:
I'd suggest moving the initialization of these variables (not the declaration) into the `beforeEach` function to ensure they are reset at the start of each test.